docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -178,6 +180,21 @@ Elements related to specifying and overriding WebLogic domain configuration:
   * `domainType`: WebLogic Deploy Tooling domain type. Legal values: WLS, RestrictedJRF, JRF. Defaults to WLS.
   * `runtimeEncryptionSecret`: The name of the Secret containing the runtime encryption password, which must be in a field named `password`. Required when `domainHomeSourceType` is set to `FromModel`.
   * `modelHome`: Location of the WebLogic Deploy Tooling model home directory, which can include model YAML files, `.properties` variable files, and application `.zip` archives. Defaults to `/u01/wdt/models`.
+  * `onlineUpdate.*`: Settings related to the online update option for Model In Image dynamic updates.
+    * `onlineUpdate.enabled`: Enable online update for model changes to a running domain. Default is `false`.
+    * `onlineUpdate.onNonDynamicChanges`:
+      Controls behavior when non-dynamic WebLogic configuration changes are
+      detected during an online update.
+      Non-dynamic changes are changes that require a domain restart to take effect.
+      Valid values are `CommitUpdateOnly` (default), `CommitUpdateAndRoll`, and `CancelUpdate`.
```

Comment:
We need to remove CancelUpdate (it no longer exists).

Reply:
Is this file generated?  I don't seem to be able to find any CanelUpdate

Reply:
This particular file isn't generated. Perhaps Tom commented on an older version?

Reply:
Still needs to be fixed.

Reply:
Fixed it.